packages/web/config/feature-flag.ts

```diff
@@ -56,17 +56,19 @@ export const Announcement:
       },
     }
   : {
-      localStorageKey: "show_mars_banner",
-      enTextOrLocalizationPath: "mars.banner.title",
+      localStorageKey: "feedback_wQ1KR7",
+      enTextOrLocalizationPath: "Help us shape the future of Osmosis.",
```

Comment:
We need to add localization to this

Reply:
It's only going to be up for about a week. Since the form is in english, syed and I agreed it's fine to just use english for now.